flagsmith-rum/manifest.json

```
@@ -0,0 +1,43 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a88f10b6-aef7-41df-979e-d70b720c6752",
+  "app_id": "flagsmith-rum",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Enriches your RUM data with your feature flags from Flagsmith",
+    "title": "Flagsmith",
+    "media": [
+      {
+        "caption": "Datadog RUM with Flagsmith Flags overview",
+        "image_url": "images/flag_rum_overview.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "Datadog RUM with Flagsmith Flags details",
+        "image_url": "images/flag_rum_details.png",
+        "media_type": "image"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::macOS",
+      "Supported OS::Windows",
+      "Category::Notification",
```

```suggestion
      "Category::Configuration & Deployment",
```

nit: perhaps this category makes more sense here